index.html

```diff
@@ -0,0 +1,99 @@
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Why_do_we_use_it" class="nav-link">Why do we use it?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Where_can_I_get_some" class="nav-link">Where can I get some?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#standard_Lorem_Ipsum" class="nav-link">The standard Lorem Ipsum passage</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <main class="mt-4" style="width: 78%; position: absolute; right: 0;" id="main-doc">
+            <section class="main-section" id="What_is_Lorem_Ipsum">
+                <header class="h2" >What is Lorem Ipsum?</header>
+                <code>What is Lorem Ipsum?</code>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                </p>
+                <ul>
+                    <li>At vero et odio </li>
+                    <li>eos et</li>
+                    <li>accusamus</li>
+                    <li>iusto</li>
+                    <li>dignissimos</li>
+                    <li>survived</li>
+                </ul>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="Where_does_it_come_from">
+                <header class="h2">Where does it come from?</header>
+                <code>Where does it come from?</code>
+                <p>
+                    Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                </p>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="Why_do_we_use_it">
```

Comment:
You are using an unused CSS selector `main-section`

Reply:
I think that it would be better to reference your original comment when the issue is repeated multiple times! it is a little bit more efficient for you! and remember to be :heart: polite ❤️ and mention the reason for the change!